### DISCUSSION TITLE
Repo shown to be 100% HTML even though there are more javascript files than HTML

### DISCUSSION THREAD

**Discussion by catfoolyou** [github-linguist/linguist#6663] [Classification] [answered]
Multiple of my repositories are getting labeled as 100% HTML, even though the repos in question have one or two HTML files an a bunch of JS and and Java files in them that are not recognized.

**Comment by lildude** [comment]
Please provide a path to one such repo so we can see and provide an explanation.

**Reply by lildude** [reply, marked as answer]
Nevermind. I've seen https://github.com/catfoolyou/Five-Nights-At-Winstons and https://github.com/catfoolyou/EaglercraftX-Client... they're only showing HTML because your JavaScript is minified which we don't count as we consider it generated code.

As for https://github.com/catfoolyou/Eagler-Online which appears to contain what could be Java files... these aren't counted because the Java files appear to all be JAR files (aka archives like zip file) and not actual raw Java.

**Reply by catfoolyou** [reply]
Ok. Now I see why its happening. Why isn't minified JS considered code?

**Reply by lildude** [reply]
It is considered code however it's considered generated code as no one writes minified code themselves and it is often third party code, like in your repos, which most people don't want counting as their own. If you really want them to count, you can use an [override](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md).

**Reply by catfoolyou** [reply]
Ok. I might use that in future projects though. Thanks.
